numpy/lib/format.py

```diff
@@ -654,7 +664,9 @@ def read_array(fp, allow_pickle=True, pickle_kwargs=None):
         If this is not a real file object, then this may take extra memory
         and time.
     allow_pickle : bool, optional
-        Whether to allow reading pickled data. Default: True
+        Whether to allow writing pickled data. Default: False
+        Switched from True to False in version 1.17.0 and 1.16.3 in response
+        to CVE-2019-6446.
```

Comment:
Perhaps `..versionchanged:: 1.17.0, 1.16.3`

Reply:
(and below)

Reply:
Maybe
```
Whether to allow writing pickled data. Default: False.

.. versionchanged:: 1.17.0, 1.16.3
    Switched from True to False in response to CVE-2019-6446.
```

Reply:
If that is acceptable, I'll make the change and merge.

Reply:
That sounds good to me, assuming sphinx is fine with the multiple versions. If not, then 1.16.3 is problem specific enough.

Reply:
I think the multiple version will work, probably everything to the right of `::` is the *version*. We will find out :)

Reply:
Turns out that the two versions version doesn't work correctly. Apparently the version ends at the first space.